src/pages/Components/Select/code.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+Import the settings and the select `scss` files.
+
+```css
+@import 'settings-tools/_all-settings';
+@import 'components/_c.select';
+```
+
+## Basic usage
+
+Apply the classes `mc-select`on the select wrapper, then `mc-select__input` on the native element of the select and `mc-select__icon` on the arrow's icon svg, as in the following example :
+
+```html
+<div class="mc-select">
+  <select class="mc-select__input">
+    <option value="" selected>Choose an option</option>
+    <option value="option1">option1</option>
+    <option value="option1">option2</option>
+    <option value="option1">option3</option>
+  </select>
+  <svg class="mc-select__icon">
+    <use xlink:href="#icon"></use>
+  </svg>
+</div>
+```
+
+<preview path="src/pages/Components/Select/previews/Select-default"></preview>
+
+
+## Behaviors
+
+### Standard select states :
+
+- `hover`
+- `focus`
+- `disabled`
+- `is-valid`
+- `is-invalid`
+
+
+<preview path="src/pages/Components/Select/previews/Select-all-states"></preview>
+
+## Default options
+ By default, you can select all options. **Each one must have a value attribute filled** which will be sent to server when the option is selected by the user. The select element hasn't placeholder, so the first option is displayed by default.
+ 
+ ```html
+<div class="mc-select">
+  <select class="mc-select__input">
+    <option value="option1">option displayed by default</option>
+    <option value="option2">option2</option>
+    <option value="option3">option3</option>
+  </select>
+</div>
+```
+
+## Preselected options
+You can choose to select an option by default with `selected` if you need it, you can also hide it with `hidden` when the user has choosen an option or simply disabled it with `disabled`.
```

Comment:
**disable** it with `disabled`.

Reply:
maybe be more precise in the terminology, "You can choose to select an option by default by applying a `selected` attribute on ..." with inline code example.